eng/install-native-dependencies.sh

```diff
@@ -14,7 +14,9 @@ elif [ "$1" = "OSX" ]; then
     if [ "$?" != "0" ]; then
         exit 1;
     fi
-    brew install icu4c openssl
+
+    engdir=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
`BASH_SOURCE` is only available in bash shells which is no longer the default on newer macOS versions. This script uses `#!/usr/bin/env sh` as the shebang so getting bash isn't guaranteed.

It'd be better to change the shebang to `#!/usr/bin/env bash` like we have in dotnet/runtime.